JASP-Desktop/analysis/analysis.cpp

```diff
@@ -26,7 +26,6 @@
 #include "dirs.h"
 #include "analyses.h"
 #include "analysisform.h"
-#include "utilities/qutils.h"
```

Comment:
Why do you remove this include?
It adds nice stuff like `<< QString` and `tq()` and more.